### DISCUSSION TITLE
LDM Notes for March 21st, 2022

### DISCUSSION THREAD

**Discussion by 333fred** [dotnet/csharplang#5949] [LDM Notes]
https://github.com/dotnet/csharplang/blob/main/meetings/2022/LDM-2022-03-21.md

## Agenda

1. file private visibility
2. Open question in semi-auto properties
3. Open question in required members

**Comment by qrli** [comment]
> We see problems with both presented options of file private and private

It sounds very like the `private protected` debate before. What you want is in fact `file & private`, while `file private` may be read as `file | private`. Maybe a new accessibility `fileprivate` would be better, at least better than `private private`. 😆
